IDE/WIN/user_settings_dtls.h

```diff
@@ -0,0 +1,98 @@
+#ifndef _WIN_USER_SETTINGS_H_
```

Comment:
You must add this file to the `IDE/WIN/include.am` so it is included in `make dist`. As-is won't  pass Jenkins.